src/ServiceContainer/ServiceProvider/ServiceContainerServiceProvider.php

```diff
@@ -150,7 +150,34 @@ public function getContainerDefinition() {
       'arguments' => array('cron'),
     );
 
-    // @todo Make it  possible to register all ctools plugins here.
+    if (module_exists('ctools')) {
+      ctools_include('plugins');
+      $info = ctools_plugin_get_plugin_type_info();
+      foreach($info as $module_name => $plugins) {
+        $services[$module_name . '.manager'] = array(
+          'class' => '\Drupal\service_container\Plugin\ContainerAwarePluginManager',
+          'arguments' => array(
+            $module_name . '.manager.internal.',
+          ),
+          'calls' => array(
+            array(
+              'setContainer',
+              array(
+                '@service_container',
+              ),
+            ),
+          ),
+        );
+
+        foreach($plugins as $plugin_type => $plugin_data) {
+          $services[$module_name . '.' . $plugin_type] = array();
```

Comment:
We should lowercase plugin type for the service and module definition. Or rather:

un-camelCase it

e.g

RenderController => render_controller

There is a handy function to camelCase and convert back in Symfony for that.